product_listing/product_listing.html

```diff
@@ -0,0 +1,231 @@
+                <p class="product--downpayment">Zero downpayment</p>
+                <div class="product-cta">
+                   <button class="product__btn">Book Now</button>
+                   <img src="/images/home-page/wishlist-black.png" alt="" class="product-wishlist">
+                </div>
+           </div>
+                
+           <div class="product--card ">
+            <img src="/images/home-page/swift.jpg" alt="swift car" class="product__img">
+            <h3 class="product--name">2013 Maruti Swift Dzire</h3>
+            <p class="product--version">VDI BS IV span </p>
+            <div class="product-spec">
+                <p class="product-info">40,212 km</p>
+                <p class="product-info">1st Owner</p>
+                <p class="product-info">Diesel</p>
+            </div>
+            <div class="product-price">
+                <p class="product-emi">₹9,000/month</p>
+                <p class="product-total">₹5,00,000</p>
+            </div>
+            <p class="product--downpayment">Zero downpayment</p>
+            <div class="product-cta">
+               <button class="product__btn">Book Now</button>
+               <img src="/images/home-page/wishlist-black.png" alt="" class="product-wishlist">
+            </div>
+       </div>
+
+       <div class="product--card ">
+        <img src="/images/home-page/swift.jpg" alt="swift car" class="product__img">
+        <h3 class="product--name">2013 Maruti Swift Dzire</h3>
+        <p class="product--version">VDI BS IV span </p>
+        <div class="product-spec">
+            <p class="product-info">40,212 km</p>
+            <p class="product-info">1st Owner</p>
+            <p class="product-info">Diesel</p>
+        </div>
+        <div class="product-price">
+            <p class="product-emi">₹9,000/month</p>
+            <p class="product-total">₹5,00,000</p>
+        </div>
+        <p class="product--downpayment">Zero downpayment</p>
+        <div class="product-cta">
+           <button class="product__btn">Book Now</button>
+           <img src="/images/home-page/wishlist-black.png" alt="" class="product-wishlist">
+        </div>
+   </div>
+
+   <div class="product--card ">
+    <img src="/images/home-page/swift.jpg" alt="swift car" class="product__img">
+    <h3 class="product--name">2013 Maruti Swift Dzire</h3>
+    <p class="product--version">VDI BS IV span </p>
+    <div class="product-spec">
+        <p class="product-info">40,212 km</p>
+        <p class="product-info">1st Owner</p>
+        <p class="product-info">Diesel</p>
+    </div>
+    <div class="product-price">
+        <p class="product-emi">₹9,000/month</p>
+        <p class="product-total">₹5,00,000</p>
+    </div>
+    <p class="product--downpayment">Zero downpayment</p>
+    <div class="product-cta">
+       <button class="product__btn">Book Now</button>
+       <img src="/images/home-page/wishlist-black.png" alt="" class="product-wishlist">
+    </div>
+</div>
+
```

Comment:
Use html formatting before pushing code on github

Reply:
ok, thank you i will do that

> Use html formatting before pushing code on github

ok, thank you i will do that